src/main/java/com/google/sps/servlets/ListOrganizationsServlet.java

```diff
@@ -48,9 +48,20 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     GivrUser currentUser = GivrUser.getLoggedInUser();
 
+    /* These statements show how to reference the parameters being received, for easier implementation of this backend */
+    if (request.getParameter("zipcode") != null) {
```

Comment:
It's good practice to remove these types of statements when submitting unless you really need it.